src/applications/appeals/download/containers/ViewAppeal.jsx

```diff
@@ -168,6 +184,7 @@ const ViewAppeal = () => {
           class="screen-only"
           onClick={window.print}
           text="Print this page for your records"
+          uswds
```

Comment:
@Mottie this might be a lingering `uswds`. Does the subtitle div need it?

Reply:
This `uswds` is added to a `va-button`, so it's fine. Expand the code up.

Reply:
Ope that's right. Was looking at the wrong beginning of code.